driver-core/src/test/unit/com/mongodb/client/model/AggregatesSpecification.groovy

```diff
@@ -114,17 +118,10 @@ class AggregatesSpecification extends Specification {
                      first('first', '$quantity'),
                      last('last', '$quantity'),
                      push('all', '$quantity'),
-                     addToSet('unique', '$quantity'))) == groupDocument
-
-        toBson(group(null,
```

Comment:
This is testing the overloaded group method that takes a List<BsonField>, so it shouldn't be removed